apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -380,6 +382,18 @@ def _load_module(path: str, course: Course):
     """
     filepath = Path(path) / "module.yaml"
     data = _load_yaml(filepath)
+
+    schemapath = "../../apps/librelingo_yaml_loader/schemas/module.json"
```

Comment:
> Is it alright to have schemas located in apps/librelingo_yaml_loader and is there any way to get around hard coding this path to load the schema json files from? I considered it being within the courses directory but I don't think there is a way for it to be there without duplicating the same schema file across multiple courses

regarding this, Python has a built in feature for this and Poetry makes sure that these things are properly set up in the final package, so this solution should work:

https://docs.python.org/3/library/pkgutil.html#pkgutil.get_data

or you could use `pkg_resources` for this